common/lsp/lsp-protocol.yaml

```diff
@@ -228,3 +228,13 @@ InlayHint:
   tooltip?: string
   paddingLeft?: boolean
   paddingRight?: boolean
+
+# -- textDcoument/rename
+
+OptionalVersionedTextDocumentIdentifier:
+  <: TextDocumentIdentifier
+  version?: integer
+
+TextDocumentEdit:
+  textDocument: OptionalVersionedTextDocumentIdentifier
+  edits: object  # TextEdit[]
```

Comment:
```suggestion
  edits+: TextEdit
```

Reply:
Testing the github patch suggest feature :)

Reply:
Accepted, I guess if it passes tests it should be good right?